src/main/kotlin/org/rust/ide/refactoring/RsImportOptimizer.kt

```diff
@@ -71,14 +76,26 @@ class RsImportOptimizer : ImportOptimizer {
             return true
         }
 
+        private fun removeUseSpeck(useSpeck: RsUseSpeck) {
```

Comment:
These methods are slightly inconsistent now. `removeCurlyBraces` returns `Boolean`, but `removeUseSpeck` returns `Unit`. Let's try using the same style for them

Reply:
Thanks, fixed, now `removeUseSpeckIfEmpty` also returns `Boolean`